### PR TITLE
removed pandas dependency and cleaned up filenames 

### PR DESCRIPTION
**Pull request by jspaaks**
- removed pandas dependency 
- cleaned up filenames 
- also added erroring on missing EXPERIMENT
- added docker-compose file to start the model generation

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -5,11 +5,14 @@ services:
   server:
     build:
       context: ./ui
+    image: ${EXPERIMENT}/server
```
Comment (jspaaks) [inline docker-compose.yml]: maybe we don't need $EXPERIMENT in the ui part. What about having ``evidence/server``?

Comment (sverhoeven) [reply docker-compose.yml]: The server service mounts the experiment data folder.
Don't think we need to name the image after the experiment as the image does not contain any experiment specific things.

File: generate-model.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+version: '3'
```
Comment (sverhoeven) [inline generate-model.yml:1]: Having a file that is not documented seems bad form, either document how to use it or move it to another PR.

File: README.md [hunk 3]
```diff
@@ -48,6 +48,8 @@ docker run -ti --volume ${PWD}/model-generation/notebooks:/data/notebooks \
 
 ### Running non-interactively
 
+with ``docker``:
```
Comment (sverhoeven) [inline README.md]: Can we give a single method instead of 2? As a user I don't care it can be done in 2 ways I want to know THE way.

File: docker-compose.yml (hunk 1 above)
Comment (sverhoeven) [reply docker-compose.yml]: Also this requires you to rebuild the images when you switch experiment, which should not be needed or wanted.

**Review by sverhoeven — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (jspaaks) [reply docker-compose.yml]: Indeed. Removed EXPERIMENT from image names for ui and model generation. Replaced with ``evidence``.

File: README.md (hunk 3 above)
Comment (sverhoeven) [reply README.md]: As the EXPERIMENT unset warning is only shown using docker-compose. I suggest to keep that one.

Comment (jspaaks) [reply README.md]: Agree but can we do this in a later PR?

**Comment by sverhoeven** [comment]
Nice, after building, running and starting ui I was able to see the doc2vec api working.

I did get warning running `docker-compose --file generate-model.yml run --user $(id -u):$(id -g) generate-model` a second time that I should do `docker-compose  --file generate-model.yml  build` or add `--build`.  Either option should be added to README

File: README.md (hunk 3 above)
Comment (sverhoeven) [reply README.md]: well you added the second way here, so I would like to resolve it here aswell

**Comment by sverhoeven** [comment]
This PR also solves the last issue raised by @meiertgrootes in https://github.com/ADAH-EviDENce/evidence/pull/95#pullrequestreview-451359676 as here the output of model generation is correctly tied to the input of the frontend.

**Review by sverhoeven — CHANGES_REQUESTED** [review]
Please fix the single running non-interactively command.

**Comment by jspaaks** [comment]
OK, so the pandas dependency was reintroduced by merging master into this but that's not a problem I'm gonna solve a second time.

**Comment by jspaaks** [comment]
OK, so the pandas dependency was reintroduced by merging master into this but that's not a problem I'm gonna solve a second time. This is exactly why we should not have so many open PRs.

**Review by sverhoeven — APPROVED** [review]
Thanks great seeing the README becoming better and better.

**Comment by jspaaks** [comment]
Thank you Stefan for reviewing, and staying positive
